src/server/ui.js

```diff
@@ -330,8 +349,15 @@ class ServerUI extends React.Component<ServerPropsType, ServerStateType> {
         <Text style={styles.welcomeMessage}>done</Text>
         Haul running at port {this.props.port}
         <Text style={styles.progressContainer}>
-          {this._makeProgressBar('ios', 'iOS', 4)}
-          {this._makeProgressBar('android', 'Android')}
+          {Object.keys(this.state.compilationProgress)
+            .sort()
+            .map(platform => {
+              return this._makeProgressBar(
+                platform,
+                platform,
+                7 - platform.length > 0 ? 7 - platform.length : 0
```

Comment:
Future reference: we'll need to make this more robust in case we have a platform longer than 7 characters :p

Reply:
If the platform is longer than 7 it wont die, it just wont align the progress bars.  -- We could push that logic up the tree, to have state to calculate the longest platform name and pass it down the tree.  I didn't think it was worth it right now since I don't think any of the major platforms are > 7 chars.

Reply:
Yea, this is rather something for a followup if someone comes up with a weird use case ;).